app/views/curation_concerns/base/_member.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr class="<%= dom_class(member) %> attributes">
+  <td class="thumbnail">
+    <% if member.reader? %>
```

Comment:
Yeah I saw that too.  I was holding on the tests until I know if this is an acceptable approach or not.